ingestors/kafka/src/main/scala/hydra/kafka/model/TopicMetadata.scala

```diff
@@ -77,6 +77,7 @@ object TopicMetadataV2 {
       .rethrow
       .flatMap {
         case (k: GenericRecord, v: GenericRecord) => Monad[F].pure((k, Option(v), headers))
+        case(k: GenericRecord, None)  => Monad[F].pure((k, None, headers))
```

Comment:
This isn't a request for a change, but a nice to know that you can also write this as:
`(k, None, headers).pure[F]`